src/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs

```diff
@@ -71,6 +63,79 @@ public void SslStream_StreamToStream_Authentication_IncorrectServerName_Fail()
             }
         }
 
+        [Fact]
+        public void SslStream_StreamToStream_Back2Back_ClientWrite_Success()
+        {
+            SslStream clientSSLStream = null;
+            SslStream ServerSSLStream = null;
+
+            try
+            {
+                GetServerClientSSLStream(out clientSSLStream, out ServerSSLStream);
+                clientSSLStream.Write(sampleMsg);
+                clientSSLStream.Write(sampleMsg);
+            }
```

Comment:
And we don't want to verify that the write actually happened, e.g. by reading the data from the server stream and verifying the second write wasn't a nop for example?